### DISCUSSION TITLE
[Feature] View Original HEIF Image in Safari

### DISCUSSION THREAD

**Discussion by Joonga** [immich-app/immich#8961] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When viewing HEIF/HEIC images on the web, the images don't appear in the highest resolution and is very evident when zooming in. This is only evident when viewing HEIF images (almost all images taken from an iPhone) and I tend to export RAWs to HEIF before uploading to my server. I can see that most web browsers do not support HEIF however Safari does so I think viewing original HEIF/HEIC images should be a feature on Safari, and then other browsers could be added later (if they ever add support). Maybe a similar thing could be done with JPEG XL as Safari also supports those too while other browsers don't by default. 

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by nicosemp** [comment]
HEIC/HEIF image support has been added a while ago, I guess this discussion can be closed

**Reply by Joonga** [reply]
Yep!
